src/simulator_core/entities/assets/demand_cluster.py

```diff
@@ -29,50 +54,159 @@ def __init__(
             self,
             asset_name: str,
             asset_id: str,
-            panda_pipes_net: pandapipesNet
+            pandapipe_net: pandapipesNet
     ):
-        """Initialize a ProductionCluster object.
+        """Initialize a DemandCluster object.
 
         :param str asset_name: The name of the asset.
         :param str asset_id: The unique identifier of the asset.
         :param PandapipesNet pandapipe_net: Pandapipes network object to register asset to.
         """
-        super().__init__(asset_name, asset_id, panda_pipes_net)
+        super().__init__(asset_name=asset_name, asset_id=asset_id, pandapipe_net=pandapipe_net)
+
+        self.setpoints = dict()
+        self.parameters = dict()
+        self.states = dict()
+
+        self.internal_diameter = DEFAULT_DIAMETER
```

Comment:
I would make this _internal_diameter as with the production_cluster? Or is this parameter relevant for the demand cluster?